src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -1,42 +1,34 @@
-package roomescape;
+package roomescape.controller;
 
 import lombok.RequiredArgsConstructor;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.*;
+import roomescape.domain.Reservation;
+import roomescape.service.ReservationService;
 
 import java.net.URI;
 import java.util.List;
 
 @Controller
 @RequiredArgsConstructor
+@RequestMapping("/reservations")
 public class ReservationController {
 
     private final ReservationService reservationService;
 
-    @GetMapping("/")
-    public String showHomePage() {
-        return "home";
-    }
-
-    @GetMapping("/reservation")
-    public String showReservationOPage() {
-        return "reservation";
-    }
-
-    @GetMapping("/reservations")
+    @GetMapping("")
     public ResponseEntity<List<Reservation>> showReservationList() {
         return ResponseEntity.ok(reservationService.getAllReservations());
     }
 
-    @PostMapping("/reservations")
+    @PostMapping("")
```

Comment:
빈 문자열을 명시해준 이유가 있을까요?
```suggestion
    @PostMapping
```

Reply:
별 생각 없이 reservation 문자열만 지워서 이렇게 된 것 같습니다ㅜ 감사합니다 수정하도록 하겠습니다!